### ISSUE TITLE
Are there ontologies other than ncro that didn't make it to the list? 

### ISSUE THREAD

**Issue by ncro-devel** [OBOFoundry/OBOFoundry.github.io#170]
A previous version of the old master https://code.google.com/p/obo-registry/source/browse/trunk/metadata/ontologies.txt had ncro which wasn't present when the new site was made. Are there others that were missed?

**Comment by jamesaoverton** [comment]
I compared the `namespace` lines in the old file to the file listing for `ontology/` in this repo. I see two dropped entries: NCRO and APOLLO_SV. Alan has just fixed NCRO. I noticed APOLLO_SV was missing from the registry yesterday: https://github.com/OBOFoundry/purl.obolibrary.org/issues/64

The old file also has an entry for 'uberon-basic', but I think that is handled correctly in the new `uberon.md`.
